maestro/chart/values.yaml

```diff
@@ -0,0 +1,130 @@
+# Default values for maestro.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: overture/maestro
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: NodePort
+  port: 11235
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: maestro.local
+      paths: []
+
+  tls: []
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appConfig:
+  server:
+    port: 80
+  maestro:
+    kafka:
+      # REPLACE this with KAFKA host/IP
+      broker: 10.11.7.31:9092
+      indexing_topic: maestro_index_requests
+    song:
+      max_retries: 3
+      timeout_sec:
+        study: 100 # some studies take really long, +30 secs, to be downloaded
+        analysis: 5
+      indexable_study_status: PUBLISHED #,UNPUBLISHED,SUPPRESSED
+    exclusion_rules:
+      # exclusion rules file name
+      file_name: exclusion-rules.yml
+
+    # elastic search server & client props
+    elasticsearch:
+      # REPLACE this with Elasticsearch host/IP
+      cluster_nodes: 10.11.7.31:9200
+      indexes:
+        file_centric:
+          alias: file_centric
+      client:
+        # this is to control the number of documents per bulk request in elasticsearch
+        docs_per_bulk_req_max: 5000
+        # max time to wait for a connection to be established
+        connection_timeout: 5000
+        # max time to wait on idle connection (no data flow)
+        socket_timeout: 10000
+        retry:
+          max_attempts: 3
+          wait_duration_millis: 500
+
+    # Genome file repositories props
+    repositories:
```

Comment:
pls remove the unused values (values that doesn't affect the generated Kubernetes yaml file)

Reply:
i don't understand what you refer to by unused values here, all values are read by the spring application